ios/FirebaseContinue/Example/FirebaseContinue/main.m

```diff
@@ -15,8 +15,20 @@
 //
 
 @import UIKit;
+
 #import "FCNAppDelegate.h"
 
+/**
+ * Important Note: This is not a working example of using Firebase Continue.
+ *
+ * The Firebase Continue for iOS library is planned to eventually be available via Cocoapods
+ * (rather than being required to manually copy-paste its source into your project), so this
+ * Cocoapods-provided development environment lays the groundwork for that.
+ *
+ * For an actual, working sample of Firebase Continue, see the samples/ios subdirectory
+ * from the root of the repository at:
+ * https://github.com/firebase/firebase-continue/tree/master/samples/ios
+ */
 int main(int argc, char * argv[])
```

Comment:
Done, thanks! These files were autogenerated by Cocoapods, so I should have spent the time cleaning them up. Appreciate you noticing these issues!